lld/test/ELF/x86-64-dwarf5-64-debug-names-type-comdat.test

```diff
@@ -0,0 +1,18 @@
+// REQUIRES: x86
```

Comment:
We should place the DWARF64 test into the other file.

https://maskray.me/blog/2021-08-08-toolchain-testing#the-test-checks-at-the-wrong-layer#i-dont-know-an-existing-test-can-be-enhanced applies